src/InfoViz/Native/MutualInformationDiagram/index.js

```diff
@@ -139,7 +141,23 @@ function informationDiagram(publicAPI, model) {
     }
   };
 
-  publicAPI.updateStatusBarText = msg => d3.select(model.container).select('span.status-bar-text').text(msg);
+  publicAPI.updateStatusBarText = (msg) => d3.select(model.container).select('span.status-bar-text').text(msg);
+
+  publicAPI.selectStatusBarText = () => {
+    // select text so user can press ctrl-c if desired.
+    if (model.statusBarVisible) {
+      // https://www.sitepoint.com/javascript-copy-to-clipboard/
+      const range = document.createRange();
+      range.selectNode(d3.select(model.container).select('span.status-bar-text').node());
+      window.getSelection().removeAllRanges();
+      window.getSelection().addRange(range);
+      // Copy-to-clipboard doesn't work unless status bar text is an 'input':
+      // document.execCommand('copy');
```

Comment:
I tried copy-paste into input widgets on Edge, Chrome and Firefox, and they all seem to handle the special characters. So might be worth trying. Do you think destroying the user's clipboard is acceptable in this case?